index.bs

```diff
@@ -67,10 +67,8 @@ by exposing the following sets of properties to the API:
 * Allow the website to exit Picture-in-Picture mode.
 * Allow the website to check if Picture-in-Picture mode can be triggered.
 
-The proposed Picture-in-Picture API is very similar to [[Fullscreen]] as they
-have similar properties. The API only applies to
-{{HTMLVideoElement}} at the moment but is meant to be
-extensible.
+The Picture-in-Picture API is very similar to [[Fullscreen]] as they
+have similar properties.
```

Comment:
I don't think it's necessary to say this... I'm worried that anyone looking at the details of the two specifications might disagree (or might lead to confusion). Let's just treat them as different things.  

```suggestion
```